arangod/Agency/AgencyFeature.cpp

```diff
@@ -168,6 +170,28 @@ regular cluster, which needs to handle only a part of the overall load.)");
                          arangodb::options::Flags::DefaultNoComponents,
                          arangodb::options::Flags::OnAgent));
 
+  options
+      ->addOption(
+          "--agency.supervision-delay-add-follower",
+          "delay in supervision, before an AddFollower job is executed [s]",
+          new UInt64Parameter(&_supervisionDelayAddFollower),
```

Comment:
Also note that this parameter is a uint64_t, whereas other time parameters in this file (e.g. a few lines above) are of type double. This is at least inconsistent.

Reply:
This is intentional. I had `double` first, but this causes more trouble inside with types and chrono, so I changed this to `uint64_t`.